packages/fxa-settings/src/components/LocaleToggle/index.test.tsx

```diff
@@ -141,14 +162,17 @@ describe('LocaleToggle', () => {
       currentLocale: 'en',
       availableLocales: [],
       switchLocale: mockSwitchLocale,
+      clearLocalePreference: mockClearLocalePreference,
+      isUsingBrowserDefault: false,
       isLoading: false
     });
 
     const { rerender } = renderWithLocalizationProvider(<LocaleToggle />);
 
     let select = screen.getByTestId('locale-select');
     expect(select).toBeInTheDocument();
-    expect(select.children).toHaveLength(0);
+    expect(select.children).toHaveLength(1);
+    expect(screen.getByTestId('locale-option-browser-default')).toBeInTheDocument();
```

Comment:
We've been trying to avoid `test-id`s where we can since , I think we can just check for the browser text? Note from Jest docs:

> In the spirit of [the guiding principles](https://testing-library.com/docs/guiding-principles), it is recommended to use this only after the other queries don't work for your use case. Using data-testid attributes do not resemble how your software is used and should be avoided if possible. That said, they are way better than querying based on DOM structure or styling css class names. Learn more about data-testids from the blog post ["Making your UI tests resilient to change"](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change)